rmf_site_format/src/sdf.rs

```diff
@@ -781,40 +821,6 @@ impl Site {
             // from full joint with wheel torques to just kinematic simulation for whole robots.
         }
 
-        // Spawn the robots now
-        // TODO(luca) use robot properties and export this like other meshes
-        // instead of including a tag
-        for location in self.navigation.guided.locations.values() {
-            let Some(robot) = location.tags.0.iter().find_map(|l| l.spawn_robot()) else {
-                continue;
-            };
-            // For now all robots are just Search type
-            let AssetSource::Search(ref robot_type) = robot.source else {
-                continue;
-            };
-            let Some(level) = self
-                .levels
-                .values()
-                .find(|l| l.anchors.get(&location.anchor.0).is_some())
-            else {
-                // TODO(luca) this would fail if the robot was on a site anchor
-                continue;
-            };
-            // Get the location
-            let anchor = get_anchor(location.anchor.0)?;
-            let tf = anchor.translation_for_category(Category::Level);
-            let pose = Pose {
-                trans: [tf[0], tf[1], level.properties.elevation.0],
-                ..Default::default()
-            };
-            world.include.push(SdfWorldInclude {
-                uri: "model://".to_string() + robot_type,
-                name: Some(robot.name.0.clone()),
-                pose: Some(pose.to_sdf()),
-                r#static: Some(robot.is_static.0),
-                ..Default::default()
-            });
-        }
```

Comment:
Note, this will break SDF export since going forward maps will not have robots in them anymore, or to be more specific, robots will be there but they won't have robot properties (before we were just including the whole SDF tag).
This is a fairly complex topic though, I'll add it to the followup PR and not hold it as a blocker for this PR.